tests/test_chromatic.py

```diff
@@ -2547,6 +2706,9 @@ def test_atredshift():
     image1 = final1.drawImage(nx=64, ny=64, scale=0.2, bandpass=bandpass)
     image2 = final2.drawImage(nx=64, ny=64, scale=0.2, bandpass=bandpass)
     np.testing.assert_allclose(image1.array, image2.array, atol=1.e-6)
+    assert gal1.redshift == 0.
```

Comment:
Not this PR, but appears that this function is `atredshift` but called as `atRedshift` in the ifmain block.